StackExchange.Redis/StackExchange/Redis/ConfigurationOptions.cs

```diff
@@ -208,6 +210,12 @@ public CommandMap CommandMap
         }
 
         /// <summary>
+        /// The retry policy to be used for connection reconnects
+        /// </summary>
+        public IReconnectRetryPolicy ReconnectRetryPolicy { get { return reconnectRetryPolicy ?? new LinearRetry(ConnectTimeout); } set { reconnectRetryPolicy = value; } }

```

Comment:
Since this creates an object for the null case, the format should change a bit. Right now, it'll spin up a new `LinearRetry` on each property fetch - should capture it to the private when null, e.g. in the `get`:
```c#
return reconnectRetryPolicy ?? (reconnectRetryPolicy = new LinearRetry(ConnectTimeout));
```

Sidenote: I don't recommend the `static` default option, since it'd be a bad long-term assumption of a stateless retry policy. Would just approach this with the tweak above.